packages/webview_flutter/webview_flutter_android/lib/src/v4/src/android_webview_controller.dart

```diff
@@ -259,7 +259,7 @@ class AndroidWebViewController extends PlatformWebViewController {
     return Point<int>(position.dx.round(), position.dy.round());
   }
 
-  @override
+  /// Whether to enable the platform's webview content debugging tools.
   Future<void> enableDebugging(bool enabled) =>
```

Comment:
A test for this method already exists at https://github.com/flutter/plugins/blob/v4_webview/packages/webview_flutter/webview_flutter_android/test/v4/android_webview_controller_test.dart#L670. The method is now just not an override.